docs/getting-started/master.adoc

```diff
@@ -8,12 +8,12 @@ include::topics/templates/document-attributes.adoc[]
 = {getting-started-guide-name}
 
 [.lead]
-This guide takes you through modern application development with {ProductName}. It shows you how to create and deploy example cloud-native applications called Boosters to OpenShift using the {launcher} application. These example applications can serve as the foundation for your actual cloud-native microservices since they can be updated and redeployed using the same deployment process detailed in this guide. 
+This guide takes you through modern application development with {ProductName}. It shows you how to create and deploy example cloud-native applications called Boosters to OpenShift using the {launcher} application. These example applications can serve as the foundation for your actual cloud-native microservices since they can be updated and redeployed using the same deployment process detailed in this guide.
 
 
 include::topics/concept-getting-started-intro.adoc[leveloffset=+1]
 
-include::topics/proc-configure-launcher-sso-realm.adoc[leveloffset=+1]
+//include::topics/proc-configure-launcher-sso-realm.adoc[leveloffset=+1]
```

Comment:
@inoxx03 this is unrelated to the SSO mission. This is about the Federated Identity access for the launcher. Do you mind uncommenting this line?

Reply:
@rhoads-zach done, I uncommented the procedure topic.